src/gwt/www/docs/keyboard.htm

```diff
@@ -886,7 +886,7 @@ <h2 id="views">Views</h2>
     <tr>
       <td>Add source column</td>
       <td>Ctrl+F7</td>
-      <td>Ctrl+F7</td>
+      <td>Command+F7</td>
```

Comment:
Nit: Should we be consistent and use `Cmd+F7` instead of typing out Command?

Reply:
This page is primarily used by screen-reader users (it replaces the inaccessible shortcut popup when screen-reader mode is enabled). "Command" is pronounced correctly with screen readers, but "Cmd" can have issues.

Reply:
Oops, I see I have some that are "cmd", let me fix that! :-)